components/prism-bqn.js

```diff
@@ -0,0 +1,65 @@
+Prism.languages.bqn = {
+	'shebang': {
+		pattern: /^#![ \t]*\/.*/,
+		alias: 'important',
+		greedy: true
+	},
+	'comment': {
+		pattern: /#.*/,
+		greedy: true
+	},
+	'string-literal': {
+		pattern: /"(?:[^"]|"")*"/,
+		greedy: true,
+		alias: 'string'
+	},
+	'character-literal': {
+		pattern: /'.'/,
+		greedy: true,
+		alias: 'string'
+	},
+	'function': {
+		pattern: /•[\w¯.∞π]+[\w¯.∞π]*/i
+	},
```

Comment:
```suggestion
	'function': /•[\w¯.∞π]+[\w¯.∞π]*/i,
```

Reply:
Also, because of the `i` flag, this regex also matches "Π". Is that intentional?

Reply:
It was not, just pushed another commit to rectify this in both function names as well as words. 'Π' is never valid outside of a char/string literal.